index.js

```diff
@@ -68,7 +78,18 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  if (movies.length < 1) {
+    throw "No Movies!";
+  }
+  let result = null;
+  return movies.find((movie) => {
+    if (movie.imdbID === id) {
+      result = movie;
+      return result;
+    }
```

Comment:
So this would look like so:
`let result = movies.find((movie) => {
    if (movie.imdbID === id) {
      return result;
    })`
`return result ? result : null;`